src/copy.jl

```diff
@@ -35,29 +35,47 @@ Runs [`execute`](@ref execute(::Connection, ::String)) on `copyin`'s query, then
 
 All other arguments are passed through to the `execute` call for the initial query.
 """
-function execute(jl_conn::Connection, copy::CopyIn, args...; throw_error=true, kwargs...)
+function execute(
+    jl_conn::Connection,
+    copy::CopyIn,
+    parameters=nothing;
+    throw_error=true,
+    kwargs...,
+)
     level = throw_error ? error : warn
```

Comment:
not related to this PR but wouldn't `log` be a better name for this?

Reply:
Could be `log_function`

People might expect `log` to be an existing log function